client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/SonarLintBackend.java

```diff
@@ -53,6 +53,9 @@ public interface SonarLintBackend {
   @JsonDelegate
   RulesService getRulesService();
 
+  @JsonDelegate
+  BindingSuggestionService getBindingSuggestionService();
```

Comment:
I wonder if this is not too granular as a service. We can always change later, but maybe we could already name it simply `BindingService`?

Reply:
Good point; Address in [078e014](https://github.com/SonarSource/sonarlint-core/pull/594/commits/078e01419b054088432516cdda1ff62c8418bd96)